torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -2206,6 +2207,7 @@ void InitXlaModuleBindings(py::module m) {
            !xtensor->CurrentDataHandle()->HasValue();
   });
   m.def("_init_xla_lazy_backend", []() {
+    absl::InitializeLog();
```

Comment:
Hmmm, I'm not confident about this change. IIRC this function errors out if it's called twice, and it is for sure called in the GPU plugin: https://github.com/openxla/xla/blob/df104c0963bf1597358dc759da3fa3d2639de03c/xla/pjrt/c/pjrt_c_api_gpu.cc#L28

If you're considering a backport, a more conservative change would be to add the initialization for TPU only sine we're sure `libtpu` does not init logging: https://github.com/pytorch/xla/blob/0b9513cb15f812252493c362ab67862bd664a54b/torch_xla/csrc/runtime/pjrt_registry.cc#L111

Reply:
sg, will update